phpstan.neon

```diff
@@ -59,6 +59,8 @@ parameters:
             # in fact, anything can be passed to the load function, it just has a bad annotation
             message: '#^Parameter \#1 \$key of method Overblog\\DataLoader\\DataLoaderInterface::load\(\) expects string#'
             path: %currentWorkingDirectory%/*
+        -
```

Comment:
Do we still call the clear method with a parameter anywhere? Ah, I see, in 14.5 version, we do. I was trying to find when did we got rid of that, but I was not successful. I would love to resolve the issue rather than ignoring it, however, I can see it is out of the scope of this pull request :shrug: